.github/workflows/todo-checker.yml

```diff
@@ -0,0 +1,25 @@
+---
+# This workflow triggers a scan of TODOs in the current pull request.
+# For each TODO that is not marked to be ignored, it posts a comment.
+# It helps ensure that no TODOs are accidentally left behind in merged PRs.
+
+name: Check TODOs via global workflow
+
+
+on:
+    issue_comment:
```

Comment:
I'd probably cut it down to just `types: [created]` otherwise the default afaik is everything (deleted, edited, created).